src/otx/core/data/mem_cache.py

```diff
@@ -168,8 +169,9 @@ def put(
 
                 if new_page > self.mem_size:
                     self.freeze()
-                    msg = "Memory pool reaches it's limit. Cannot cache more. Freeze it."
-                    logger.warning(msg)
+                    if os.getenv("DEBUG_MODE", "false").lower() == "true":
+                        msg = "Memory pool reaches it's limit. Cannot cache more. Freeze it."
+                        logger.warning(msg)
```

Comment:
```suggestion
                    logger.debug(msg)
```
How about using logger.debug?

Reply:
Thanks! fixed - https://github.com/openvinotoolkit/training_extensions/pull/3757/commits/2ebfb98bf50221841a18eb7d0dcb306c8707145b

Reply:
ah sorry I missed `msg`. Could you add it again?

Reply:
Thanks, done - https://github.com/openvinotoolkit/training_extensions/pull/3757/commits/99fe5fc73076b2cd435c24e54dc694791aa3b489